packages/embeds/embed-react/package.json

```diff
@@ -52,7 +53,7 @@
     "vite": "^4.1.2"
   },
   "dependencies": {
-    "@calcom/embed-core": "workspace:*",
```

Comment:
Is this strictly necessary? Can we test this on self hosted to prevent conflicts?

Reply:
Yeah it's necessary and it works well. As yarn 3 has packages linking support out of the box now. 
In monorepo it would symlink the packages, so npm repository isn't used to fetch what's already there.

Reply:
I will test the self hosting flow as well

Reply:
Even though it works. It looks like yarn has a better way to solve this problem
https://yarnpkg.com/features/workspaces#publishing-workspaces
![Uploading Screenshot 2023-04-14 at 12.34.18 PM.png…]()